constants.go

```diff
@@ -48,6 +48,13 @@ const (
       <source network='{{.Network}}'/>
       <model type='virtio'/>
     </interface>
+    <console type='pty'></console>
+    <channel type='pty'>
+      <target type='virtio' name='org.qemu.guest_agent.0'/>
+    </channel>
```

Comment:
This part is needed when you want to run qemu-guest-agent in the guest. Since this binary is not present on the coreos image we are using, this device is not needed.